docs/advanced-topics/checksum-support.rst

```diff
@@ -0,0 +1,67 @@
+#########
+Checksums
+#########
+
+To help developers who want to distribute or run immutable copies of tools and workflows, Dockstore (1.9+) can provide checksums for files and Docker
+images when certain conditions are met. Currently, checksums are not accessible through the UI, but can be fetched by using various TRS
+endpoints. Keep reading to learn what is supported and how to retrieve the checksum information.
+
+File Descriptor Checksum Support
+================================
+As of 1.9, Dockstore will calculate a sha1 checksum during a refresh for every container, descriptor, and test parameter file included in a
+tool or workflow. Once the refresh is done, you must publish your entry in order to access the information via our `TRS V2 endpoints <https://dev.dockstore.net/api/static/swagger-ui/index.html#/GA4GHV20>`_.
```

Comment:
what's the plan on switching from dev to production?

Reply:
changed to future production links